src/main/resources/wufoo-sysaid-field-mapping-sample.properties

```diff
@@ -26,8 +28,8 @@ Comment\u0020or\u0020Question.fieldtype=Text
 #and still the value is empty defaults will be used.
 ############################
 sysaid.defaultfield=description
-sysaid.defaults.responsibility=1221
```

Comment:
Add comments here to explain what each of these values expect. Also, list the options available.

Example: # Status options: 1,2,3,4,5
               # 1 - New
               # ...